targets/PythonSdk/source/source/PlayFabSettings.py

```diff
@@ -0,0 +1,27 @@
+import PlayFabErrors
+
+ProductionEnvironmentURL = "https://{titleId}.playfabapi.com{methodUrl}"
+TitleId = "" # You must set this value for PlayFabSdk to work properly (Found in the Game Manager for your title, at the PlayFab Website)
+GlobalErrorHandler = None
+EntityToken = None # Internal variable used for Entity API Access (basically Entity Login)
+DeveloperSecretKey = None # You must set this value for PlayFabSdk to work properly (Found in the Game Manager for your title, at the PlayFab Website)
+ClientSessionTicket = None # This is set
+SdkVersionString = "PythonSdk-<%- sdkVersion %>";
```

Comment:
This is correctly generated here, so we should not ever suggest in our tests or examples that the customer should ever look at or touch this.
We might even hide a few things in a _internalSettings sub-object inside of settings now that I think about it.
Half of these variables aren't meant for customers to touch.